defaults/main.yml

```diff
@@ -20,6 +20,9 @@ postgresql_service_user: "{{ postgresql_admin_user }}"
 postgresql_service_group: "{{ postgresql_admin_user }}"
 postgresql_service_enabled: true
 
+# If a parmeter change requires a restart, then issue the restart?  Default to safety.
```

Comment:
Open to discussion but I personally find the language proposed below easier to understand.
```suggestion
# If set to true, any parameter changes that require a restart will automatically trigger a restart.  Default to requiring manual restart (false).
```